docs/group-user-management.rst

```diff
@@ -0,0 +1,227 @@
+
+       ``$ omero group add Lab1 --type=read-annotate``
+
+       or
+
+       ``$ omero group add Lab1 --perms='rwra--'``
+
+    b. To list all the groups and save the output for example in a CSV file:
+
+       ``$ omero group list --style csv > groups.csv``
+
+    c. To add an existing user user-1 to the Lab1 group and make him/her a group owner (the option ``--as-owner`` is not needed when adding a member), run:
+
+       ``$ omero group adduser user-1 --name=Lab1 --as-owner``
+
+    d. Let’s add trainer-1 as an owner of the group too:
+
+       ``$ omero group adduser trainer-1 --name=Lab1 --as-owner``
+
+    e. To remove user-1 from the list of owners (user-1 will still be a member of the Lab1 group):
+
+       ``$ omero user leavegroup Lab1 --name=user-1 --as-owner``
+
+    f. To remove user-1 from the Lab1 group, run:
+
+       ``$ omero group removeuser user-1 --name=Lab1``
+
+    g. To edit the Lab1 group, first determine its ID:
+
+       ``$ omero group info --group-name Lab1``
+
+       ``id \| name \| perms \| ldap \| # of owners \| # of members``
+
+       ``-----+-------+--------+-------+-------------+--------------``
+
+       ``653 \| Lab1 \| rwra-- \| False \| 0 \| 0``
+
+    h. Change the group name to ``LabN``:
+
+       ``$ omero obj update ExperimenterGroup:653 name='LabN'``
+
+    i. Let’s reset the name back to ``Lab1`` to simplify the rest of the workflow
+
+    j.  Change groups permissions to read-write:
+
+        ``$ omero group perms --perms='rwrw--' --name='Lab1'``
+
+2. Managing Users:
+
+   a. Create a new user with login name lpasteur and at the same time add this user (with first and last name ``Louis Pasteur``) to the Lab1 group:
+
+      ``$ omero user add lpasteur Louis Pasteur --group-name Lab1``
+
+   b. Let’s now add the user to another group:
+
+      ``$ omero user joingroup Lab1 --name=lpasteur``
+
+   c. To edit the user and for example add an email address, first determine the user’s ID:
+
+      ``$ omero user info --user-name lpasteur``
+
+   d. Add an email address (supposing the ID of the user were ``123``):
+
+      ``$ omero obj update Experimenter:123 email='lpasteur@demo.co.uk'``
+
+   e. Make a user inactive. User cannot be deleted but it is possible to prevent a user from logging in. For that we need to remove the user from the user group (internal OMERO group).
+
+      ``$ omero user leavegroup user --name=lpasteur``
+
+   f. To reactivate the user:
+
+      ``$ omero user joingroup user --name=lpasteur``
+
+   g. LDAP authentication. It is possible to convert non LDAP users to LDAP authentication using the command ``omero ldap setdn``. When using LDAP as an authentication backend, users when they log in will be added to the internal OMERO group called default unless they have already been added to a given group. To add a user before they have ever logged in to OMERO, first create the user (example user name is ``enoether``).
```

Comment:
```suggestion
   g. LDAP authentication. It is possible to convert non-LDAP users to LDAP authentication using the command ``omero ldap setdn``. When using LDAP as an authentication backend, users when they log in will be added to the internal OMERO group called default unless they have already been added to a given group. To add a user before they have ever logged in to OMERO, first create the user (example user name is ``enoether``).
```

Reply:
How do you use ```omero ldap setdn```? e.g. ```omero ldap setdn --name enoether ``` ?

Reply:
@will-moore I will have to defer to @joshmoore for the answer

Reply:
e.g. `omero ldap setdn true --user-name enoether `

Looks like in a refactoring `--group-id` and `--group-name` got added which make no sense here. cc: @sbesson